packages/repo/src/untyped-doc-handle.ts

```diff
@@ -81,10 +81,10 @@ export class UntypedDocHandle {
       get peers() {
         // Return all peers EXCEPT self as a Map
         const allStates = synchronizer.getAllEphemeralStates(docId)
-        const result = new Map<string, ObjectValue>()
+        const result = new Map<PeerID, ObjectValue>()
         for (const [peerId, value] of Object.entries(allStates)) {
           if (peerId !== myPeerId) {
-            result.set(peerId, value)
+            result.set(peerId as PeerID, value)
```

Comment:
This is needed because `Object.entries` looses types. I am a fan of [ts-extras](https://github.com/sindresorhus/ts-extras/blob/main/source/object-entries.ts) for this case.